tests/namedtempfile.rs

```diff
@@ -387,55 +387,3 @@ fn test_make_uds() {
 
     assert!(temp_sock.path().exists());
 }
-
-#[cfg(unix)]
-#[test]
-fn test_make_uds_conflict() {
```

Comment:
This test can presumably still exist, but less deterministically, by setting the `rand_bytes` to 1, then trying to generate 62 files. (Which will require that *some* of the threads retry unless they all get incredibly lucky, which won't happen)

Reply:
This is possible, but the number of retries will no longer be deterministic.